### PR TITLE
util: make worker test more stable

### PR DESCRIPTION
**Pull request by BusyJay**
@ngaut @siddontang @disksing PTAL

### CODE REVIEW

**Comment by ngaut** [comment]
LGTM

File: src/util/worker/mod.rs [hunk 1]
```diff
@@ -243,12 +243,17 @@ mod test {
...
         worker.schedule(50).unwrap();
         assert!(worker.is_busy());
-        thread::sleep(Duration::from_millis(60));
+        for _ in 0..100 {
+            if !worker.is_busy() {
+                break;
+            }
+            thread::sleep(Duration::from_millis(10));
+        }
         assert!(!worker.is_busy());
```
Comment (disksing) [inline src/util/worker/mod.rs]: This line can be removed.

Comment (BusyJay) [inline src/util/worker/mod.rs]: No, the for loop may exit after 1s.

**Comment by siddontang** [comment]
LGTM

**Comment by disksing** [comment]
LGTM

**Pull request merged**